cmd/pbstatuser/main.go

```diff
@@ -162,6 +166,7 @@ func checkSourceAvailabilityGCP(ctx context.Context) {
 	defer processingWG.Done()
 
 	for s := range chGcp {
+		logger.Trace().Msgf("checking GCP source availability status %s", s.SourceApplicationID)
```

Comment:
```suggestion
		logger.Trace().Msgf("Checking GCP source availability status %s", s.SourceApplicationID)
```

Ok I am giving up. :-D